packages/react/src/menu/root/useMenuRoot.ts

```diff
@@ -26,11 +26,13 @@ import { useOpenChangeComplete } from '../../utils/useOpenChangeComplete';
 import { useDirection } from '../../direction-provider/DirectionContext';
 import { useScrollLock } from '../../utils/useScrollLock';
 import {
-  type OpenChangeReason,
+  type BaseOpenChangeReason,
   translateOpenChangeReason,
 } from '../../utils/translateOpenChangeReason';
 import { ownerDocument } from '../../utils/owner';
 
+export type MenuOpenChangeReason = BaseOpenChangeReason | 'item-press' | 'cancel-open';
```

Comment:
`item-press` and `cancel-open` are already defined in BaseOpenChangeReason. `sibling-open` can be moved here instead.